src/fileioc/fileioc.asm

```diff
@@ -1393,6 +1400,51 @@ ti_ArchiveHasRoom:
 	dec	a
 	ret
 
+;-------------------------------------------------------------------------------
+ti_SetPostGCHandler:
+;Set handler for setting up the screen after a garbage collect
+; args:
+;   sp + 3 : pointer to handler. Set to 0 to use default handler (xlibc palette)
+; return:
+;   None
+	pop de
+	ex (sp),hl
+	push de
+	add hl,de
+	or a,a
+	sbc hl,de
+	jr nz,.notdefault
+	ld hl,util_post_gc_default_handler
+.notdefault:
+	ld (util_post_gc_handler),hl
+	ex hl,de
+	jp (hl)
+util_post_gc_default_handler:=$F8
```

Comment:
What's going on here? Shouldn't this label simply point to a `ret`? (And you can reuse a preexisting one elsewhere, like the one in `util_pre_gc_default_handler`.)

Reply:
$F8 points to a ret instruction.

Reply:
On your OS/boot code version, sure, but there's no guarantee it's that way for others. There's really no reason not to point to an internal `ret` instead.

Reply:
Alright, will update this routine as well as the others that use $F8 as a no-op :P

Reply:
Resolved with latest commit